hello-signing.md

```diff
@@ -0,0 +1,132 @@
+  ```bash
+  export PORT=5000
+  export REGISTRY=localhost:${PORT}
+  export REPO=${REGISTRY}/net-monitor
+  export IMAGE=${REPO}:v1
+  ```
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) for local docker operations
+- Run a local instance of the [CNCF Distribution Registry][cncf-distribution]
+  ```bash
+  docker run -d -p ${PORT}:5000 ghcr.io/oras-project/registry:latest
+  ```
+- Acquire the Notation CLI  
+Notation releases can be found at: [Notation Releases][notation-releases]  
+  ```bash
+  curl -O ~/bin/notation https://github.com/notaryproject/notation/releases/download/v0.0.1-alpha/notation-v0.0.1-alpha.1-linux-amd64
+  curl -O ~/.docker/cli-plugins/docker-notation https://github.com/notaryproject/notation/releases/download/v0.0.1-alpha/docker-notation-v0.0.1-alpha.1-linux-amd64
+  ```
+
+## Building and Pushing
+- Build and Push the `net-monitor` software
+  ```bash
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+
+  docker push $IMAGE
+  ```
+- List the image, and any associated signatures
+  ```bash
+  notation list $IMAGE
+  ```
+  At this point, the results are empty, as there are no existing signatures
+
+## Signing a Container Image
+
+To get things started quickly, the notation cli supports generating self signed key-pairs. As you automate the signing of content, you will most likely want to create and store the private keys in a key vault. (Detailed production steps will be covered later)
+
+- Generating a Private Key  
+  The following will generate an X-509 private key under the `./notary/` directory
+  ```bash
+  notation cert generate "wabbit-networks.io"
+  ```
+- Sign the `net-monitor:v1` container image
+  ```bash
+  notation sign $IMAGE
+  ```
+- List the image, and any associated signatures
+  ```bash
+  notation list $IMAGE
+  ```
+## Verify a Container Image Using Notation Signatures
+
+To avoid a Trojan Horse attack, before pulling an artifact into an environment, it is important to verify the integrity of the artifact based on the identity of an entity you trust. Notation uses a set of configured public keys to verify the content. The `notation cert generate` command automatically added the public key.
+- Attempt to verify the $IMAGE notation signature
+  ```bash
+  notation verify $IMAGE
+  ```
+  *The above verification should fail, as you haven't yet configured the keys to trust.*
+- To assure users opt-into the public keys they trust, add the key to the trusted store
+  ```bash
+  notation cert add "wabbit-networks.io" ~/.notary/wabbit-networks.io.cert
+  ```
+- Verify the `net-monitor:v1` notation signature
+  ```bash
+  notation verify $IMAGE
+  ```
+  The above now succeeds as the image is signed with a trusted public key
+
+## Simulating a Registry DNS Name
+
+Here are the additional steps to simulate a fully qualified dns name for wabbit-networks.
```

Comment:
Should "dns" be all caps here?

Reply:
I figured since it's part of a title, yes. Since the doc expert asked, I'm assuming it should be lower case. 
Changed

Reply:
I should have been clearer -- I was referring to DNS in the sentence. I think it should be all caps in both places.

Reply:
(I'd also suggest adding a line break above this title)